apis/v1alpha1/amazoncloudwatchagent_types.go

```diff
@@ -164,6 +167,9 @@ type AmazonCloudWatchAgentSpec struct {
 	// ImagePullPolicy indicates the pull policy to be used for retrieving the container image (Always, Never, IfNotPresent)
 	// +optional
 	ImagePullPolicy v1.PullPolicy `json:"imagePullPolicy,omitempty"`
+	// PrometheusConfig is the raw JSON to be used as the collector's prometheus configuration.
```

Comment:
I chose this because upstream sets their Config as JSON as well, despite them passing in a YAML value: https://github.com/open-telemetry/opentelemetry-operator/blob/v0.90.0/apis/v1alpha1/opentelemetrycollector_types.go#L179.

Reply:
I will fix the comment to say YAML instead.